virtcontainers/qemu.go

```diff
@@ -736,6 +739,10 @@ func (q *qemu) hotplugVFIODevice(device *config.VFIODev, op operation) error {
 	devID := device.ID
 
 	if op == addDevice {
+		if q.state.HotplugVFIOOnRootBus {
```

Comment:
Please add a comment about the fact that choosing the root bus here will bypass the use of a bridge. This might not be completely clear from the code.

Reply:
Done.